### ISSUE TITLE
Spectron always creates new instance of partitions of electron app

### ISSUE THREAD

**Issue by rik12s** [electron-userland/spectron#976]
Generally, when we use it manually there is a fixed path where dev team has assigned. For eg: /Application Support/<Projectname>/folder used for storing data behind the app.

![image](https://user-images.githubusercontent.com/19566020/122175725-83218280-cea1-11eb-807c-33fdc0de3fb5.png)

But every time when app is launched using spectron like: 

![image](https://user-images.githubusercontent.com/19566020/122176000-bcf28900-cea1-11eb-9c2e-f7eea0fbd6c6.png)

Data which should have been saved cannot be accessed as everytime new instance created some directory like:

/private/var/folders/4f/frtzmcsd5yv9zf9p2whcfgqc0000gr/T/.**org.chromium.Chromium.XKQ5Vc**/

So basically highlighted part is changed after every execution and creates the new same folder structure as i attached in image1.

Is there any way where i can configure to use the same partiton which were used when we manually make the app up?

Or anyway to get the information at runtime, so that i can copy few files to get the data reflected in my app immediately rather then processing everytime.
